unified_planning/model/fnode.py

```diff
@@ -62,6 +62,8 @@ def __repr__(self) -> str:
             return str(self.constant_value())
         elif self.is_fluent_exp():
             return self.fluent().name + self.get_nary_expression_string(", ", self.args)
+        elif self.is_dot():
+            return self.agent().name + self.get_nary_expression_string(", ", self.args)
```

Comment:
This looks strage, I think we simply write:
```suggestion
            return "%s.%s" %(self.agent().name, self.arg(0))
```